app/controllers/provider/sessions_controller.rb

```diff
@@ -101,4 +102,8 @@ def session_return_to
   def instantiate_sessions_presenter
     @presenter = Provider::SessionsPresenter.new(domain_account)
   end
+
+  def redirect_to_sso_params
+    redirect_to authorization_provider_bounce_path(published_authentication_providers.first.system_name) if domain_account.settings.enforce_sso? && published_authentication_providers.count == 1
```

Comment:
Suggestion to do it in 2 lines just for readability 😄 this is very subjective tho and you might disagree 👍 
```suggestion
    return if !domain_account.settings.enforce_sso? || published_authentication_providers.count != 1
    redirect_to authorization_provider_bounce_path(published_authentication_providers.first.system_name)
```